testdata/baselines/reference/submodule/conformance/constEnum4.js.diff

```diff
@@ -1,27 +1,11 @@
 --- old.constEnum4.js
 +++ new.constEnum4.js
-@@= skipped -9, +9 lines =@@
-
+@@= skipped -10, +10 lines =@@
 
  //// [constEnum4.js]
--if (1)
--    ;
--else if (2)
+ if (1)
 -    ;
--else
+ else if (2)
 -    ;
```

Comment:
I think this is a regression; now we are emitting JS that won't parse, because the tree looks like:

```
if (1)
else if (2)
else
```

missing any braces.